src/commands/functions/create.js

```diff
@@ -145,57 +153,47 @@ const formatRegistryArrayForInquirer = function (lang) {
 
 // pick template from our existing templates
 const pickTemplate = async function () {
-  inquirer.registerPrompt('autocomplete', inquirerAutocompletePrompt)
-  // doesnt scale but will be ok for now
-  const [
-    jsreg,
-    // tsreg, goreg
-  ] = [
-    'js',
-    // 'ts', 'go'
-  ].map(formatRegistryArrayForInquirer)
   const specialCommands = [
-    new inquirer.Separator(`----[Special Commands]----`),
+    new inquirer.Separator(),
     {
-      name: `*** Clone template from Github URL ***`,
+      name: `Clone template from GitHub URL`,
       value: 'url',
       short: 'gh-url',
     },
     {
-      name: `*** Report issue with, or suggest a new template ***`,
+      name: `Report issue with, or suggest a new template`,
       value: 'report',
       short: 'gh-report',
     },
+    new inquirer.Separator(),
   ]
-  const { chosentemplate } = await inquirer.prompt({
-    name: 'chosentemplate',
+  const { language } = await inquirer.prompt({
```

Comment:
This is the new prompt that will ask users to select the language before presenting the available templates.